smokeTest.js

```diff
@@ -0,0 +1,112 @@
+process.argv.forEach((val, index) => {
```

Comment:
What is this doing? The `npm run smoketest` script doesn't seem to be passing any arguments to it.

Reply:
Just logging the arguments that have been passed in, I think it can be removed